src/App.vue

```diff
@@ -23,10 +23,11 @@
 			</div>
 		</div>
 		<div class="modal-body">
-			<slot v-if="slides.length > 0" name="body">
+			<slot v-if="slideList.length > 0" name="body">
 				<transition :name="fadeDirection" mode="out-in">
 					<!-- eslint-disable-next-line vue/no-v-html -->
-					<div v-if="slides[currentSlide].type === 'inline'" :key="currentSlide" v-html="slides[currentSlide].content" />
+					<div v-if="slideList[currentSlide].type === 'inline'" :key="currentSlide" v-html="slideList[currentSlide].content" />
```

Comment:
Do you have our global eslint setup here?
This should throw, no?
Or is it max 3?

Reply:
Moved to nextcloud eslint config and `@nextcloud/vue` packages